README.md

```diff
@@ -1,70 +1,127 @@
-    --to-image-base=docker.io/abhinavdahiya/origin-cluster-version-operator:latest \
-    --to-image docker.io/abhinavdahiya/origin-release:latest
+     status	<Object>
+       status contains information about the available updates and any in-progress
+       updates.
 ```
 
-## Installing CVO and operators in cluster.
+`ClusterVersion` resource follows the established [Kubernetes pattern][kube-spec-and-status] where the `spec`
+property describes the desired state that CVO should achieve and maintain, and the `status` property is populated by the
+CVO to describe its status and the observed state.
 
-1. Use CVO `render` to render all the manifests from release-payload to a directory. Refer the [Using CVO to render the release-image locally](#using-cvo-to-render-the-release-image-locally) section for more information.
+In a typical OpenShift cluster, there will be a cluster-scoped `ClusterVersion` resource called `version`:
 
-2. Create the operators from the manifests by using `oc create -f <directory when CVO rendered manifests>`.
+```console
+$ oc get clusterversion version
+NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+version   4.11.3    True        False         85d     Cluster version is 4.11.3
+```
 
-## Using CVO to render the release-image locally
+Note that as a user or a cluster administrator, you usually do not interact with the `ClusterVersion` resource directly
+but via either the [`oc adm upgrade`][ocp-oc-adm-upgrade] CLI or the [web console][ocp-webconsole-upgrades].
 
-1. Run the following command to get render the release-payload contents to `/tmp/cvo`
+## Understanding Upgrades
 
-```sh
-podman run --rm -ti \
-    -v /tmp/cvo:/tmp/cvo:z \
-    <release image> \
-        render \
-        --output-dir=/tmp/cvo \
-        --release-image="<release image>"
-```
+> :bulb: This section is only a high-level overview. See the [Update Process][dev-guide-upgrade-workflow] and
+> [Reconciliation][dev-guide-reconciliation] documents in the [dev-guide][dev-guide] for more details.
 
-`<release image>` can be personal release image generated using [this](#building-release-image-using-local-cvo) or Origin's release image like `registry.ci.openshift.org/openshift/origin-release:v4.0`.
+The Cluster Version Operator continuously fetches information about upgrade paths for the configured channel from the
+OpenShift Update Service (OSUS). It stores the recommended update options in the `status.availableUpdates` field of
+its `ClusterVersion` resource.
 
-## Running CVO tests
+The intent to upgrade the cluster to another version is expressed by storing the desired version in
+the `spec. desiredUpdate` field. When `spec.desiredUpdate` does not match the current cluster version, CVO will start
+updating the cluster. It downloads the release payload image, validates it, and [systematically reconciles]
+[dev-guide-reconciliation] the Cluster Operator resources to match the updated manifests delivered in the release
```

Comment:
nit: GitHub doesn't like breaking the text and anchor-ref across a newline:

![image](https://user-images.githubusercontent.com/209920/205720789-25cd4786-619a-4aa3-a8ca-218a985731b1.png)